core/domain/app_feedback_report_domain_test.py

```diff
@@ -243,6 +309,30 @@ def test_report_scrubber_id_is_invalid_validation_fails(self) -> None:
             'The scrubbed_by user id \'%s\' is invalid.' % (
                 self.android_report_obj.scrubbed_by))
 
+    def test_report_scrubber_id_is_bot_id_validation_passes(
```

Comment:
Can you please update the test name, according to this format:
``test_{{action}}_with_{{with_condition}}_{{has_expected_outcome}}``

Reply:
done